samples/src/jvmMain/kotlin/okio/samples/BitmapEncoder.kt

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2018 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okio.samples
+
+import okio.BufferedSink
+import okio.buffer
+import okio.sink
+import java.io.File
+import java.io.IOException
+import kotlin.math.hypot
+
+class KotlinBitmapEncoder {
+  class Bitmap(
+    private val pixels: Array<IntArray>
+  ) {
+    fun width(): Int = pixels[0].size
```

Comment:
These want to be vals with `get()` methods, right?

Reply:
Yep yep, good call! Will fix.

Reply:
I actually went with computed vals, since `pixels` never change.